cactusbot/cached/caching.py

```diff
@@ -0,0 +1,18 @@
+class CacheUtils(object):
+
+    # You have found MysticalMage's code, turn back now, no good lies ahead
+    def in_cache_followers(user):
+        """ Returns if user is in the cache. """
+        follower_cache = open("caches/CacheFollowers.txt", "r")
```

Comment:
A context manager would be cleaner here, and would eliminate the later need for `.close()`.
```python
with open("filename", 'r') as file:
    print(file.read())
```